### PR TITLE
Remove config cat flag for read replicas

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

File: apps/studio/pages/projects.tsx [hunk 1]
```diff
@@ -11,13 +11,17 @@ import { useAutoProjectsPrefetch } from 'data/projects/projects-query'
...
 
 const ProjectsPage: NextPageWithLayout = () => {
   const router = useRouter()
   const [search, setSearch] = useState('')
   const { data: organizations, isError, isSuccess } = useOrganizationsQuery()
   useAutoProjectsPrefetch()
 
+  const data = useProfile()
+  console.log(data)
+
```
Comment (alaister) [inline apps/studio/pages/projects.tsx:24]: Did this slip through? ;)

Comment (joshenlim) [reply apps/studio/pages/projects.tsx:24]: oooooooh

Comment (joshenlim) [reply apps/studio/pages/projects.tsx]: thanks so much 🙏 forgot to stash the previous code changes while i was debugging something haha

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**